src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,57 @@
+.stopwatch {
+  position: relative;
+  width: $watch-size;
+  height: $watch-size;
+  border: $border-size dotted $border-color;
+  border-radius: $circle;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__middle {
+    width: $middle-size;
+    height: $middle-size;
+    background-color: $middle-color;
+    border-radius: $circle;
+  }
+  &__minutes-hand {
+    position: absolute;
+    width: $hand-minutes-width;
+    height: $hand-minutes-height;
+    background-color: $hand-minutes-color;
+    bottom: 50%;
+    left: calc(50% - (#{$hand-minutes-width} / 2));
+    z-index: -1;
```

Comment:
It's a good practice to group related properties together. Here, position, bottom, left, and z-index properties are related as they control the positioning of the element. You may want to consider grouping them together in your code for better readability.